pkg/ipc/ipc.go

```diff
@@ -376,14 +387,32 @@ func (env *Env) parseOutput(p *prog.Prog) (*ProgInfo, error) {
 	return info, nil
 }
 
+func dedupCover(rawCover []uint32) []uint32 {
```

Comment:
Could this package just return the coverage from executor as is and don't differentiate between raw/non-raw cover?
The main user of this is syz-fuzzer and it already uses cover.Merge to combine coverage from several attempts and cover.Merge already effectively does what this function do.

Reply:
Yes, that sounds reasonable. I've updated the code, now it's simpler.